File: backend/src/main/java/com/backend/petplace/domain/place/projection/PlaceSearchRow.java
package com.backend.petplace.domain.place.projection;

public interface PlaceSearchRow {
  Long getId();
  String getName();
  String getCategory2();
  Double getLatitude();
  Double getLongitude();
  Integer getDistanceMeters();
  Double getAverageRating();
  String getAddress();

}

File: backend/src/main/java/com/backend/petplace/domain/place/projection/PlaceSearchRow.kt
package com.backend.petplace.domain.place.projection

interface PlaceSearchRow {
    val id: Long
    val name: String
    val category2: String
    val latitude: Double
    val longitude: Double
    val distanceMeters: Int
    val averageRating: Double?
    val address: String
}
